src/codec.rs

```diff
@@ -43,20 +49,78 @@ impl Codec {
             Self::Cer => crate::cer::encode(value),
             Self::Der => crate::der::encode(value),
             Self::Uper => crate::uper::encode(value),
+            Self::Jer => Err(crate::error::EncodeError::from_kind(
+                crate::error::EncodeErrorKind::Custom {
+                    msg: "JER is a text-based encoding. Call `Codec::encode_to_string` instead."
+                        .into(),
+                },
+                Codec::Jer,
+            )),
```

Comment:
Should this be the behaviour? Why not return the bytes of text based encoding. `encode_to_string`'s behaviour makes sense, but you can represent strings as bytes.

Reply:
You have a point. I made the appropriate changes.